crates/fm/src/file_reader.rs

```diff
@@ -23,6 +23,10 @@ cfg_if::cfg_if! {
 
         }
 
+        pub(super) fn is_stdlib_asset(path: &Path) -> bool {
+            path.starts_with("std\\")
```

Comment:
if I'm looking at the code correctly, this is wrong. The `std\\` would be for windows, not wasm

Reply:
Oof, my bad. I must have misread the cfg statements. Fixed in latest commit.